pkg/netpol/connlist/exposure_map.go

```diff
@@ -71,17 +71,21 @@ func (ex *exposureMaps) addPeerXgressEntireClusterExp(pe *eval.PolicyEngine, pee
 	return nil
 }
 
-// addNewEntry adds a new entry to the relevant ingress/egress map for the given peer
-func (ex *exposureMaps) addNewEntry(peer Peer, isProtected, isIngress bool) {
+// checkExistanceAndAddNewEntry adds a new entry to the relevant ingress/egress map for the given peer
```

Comment:
update comment: `adds a new entry to the relevant ingress/egress map for the given peer, if the peer's key is not in the map`

Reply:
done